index.md

```diff
@@ -5,8 +5,8 @@ description: Vanilla OS is an Immutable Linux-based distribution which aims to p
 <div class="hero hero-center hero-main">
     <div class="container hero-wrapper">
         <div class="hero-image animate__animated animate__fadeIn" data-scroll>
-            <img src="/assets/images/home-laptop.svg" alt="Vanilla OS - Showcase" class="on-light" />
-            <img src="/assets/images/home-laptop-dark.svg" alt="Vanilla OS - Showcase" class="on-dark" />
+            <img src="/assets/images/home-laptop.svg" class="on-light" alt="" />
```

Comment:
It's not recommended empty alt

Reply:
Perfect timing, I was suggesting the same in Discord

Reply:
Empty alt texts should only be used when the images are purely used for decoration: https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
But it's a website logo, is necessity

Reply:
I think they're decorative, they don't add much if anything to the website information-wise for screen readers and the like. They're purely for showcasing what the desktop looks like.

Reply:
Alright. In that case I'll leave it as is